packs/libcloud/actions/start_vm.py

```diff
@@ -18,6 +18,6 @@ def run(self, credentials, vm_id):
         if status is True:
             self.logger.info('Successfully started node "%s"' % (node))
         else:
-            self.logger.rror('Failed to start node "%s"' % (node))
+            self.logger.error('Failed to start node "%s"' % (node))
```

Comment:
I fix a bug in the existing pylint script (previously we didn't add the right path to st2 components in `PYTHONPATH`) so we didn't detect any logging related issues.